variables.tf

```diff
@@ -65,6 +65,14 @@ variable "tags" {
   description = "Tags to be assigned to the S3 bucket and the CloudFront distribution"
 }
 
+
+variable "trusted_key_groups" {
+  type    = list(string)
```

Comment:
For newer attributes, we always pass objects, not just specific attributes, i.e. this should be a list of object with requires attribute.